### ISSUE TITLE
Precision errors in statespace testing on 32-bit

### ISSUE THREAD

**Issue by matthew-brett** [statsmodels/statsmodels#3067]
Building and testing 32-bit wheels for 0.8.0rc1 - a couple of test errors:

```
======================================================================
FAIL: statsmodels.tsa.statespace.tests.test_mlemodel.test_fit_misc
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/venv/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/venv/local/lib/python2.7/site-packages/statsmodels/tsa/statespace/tests/test_mlemodel.py", line 196, in test_fit_misc
    assert_almost_equal(res_params, [0,0], 6)
  File "/venv/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 452, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/venv/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 812, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/venv/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 645, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 6 decimals
(mismatch 50.0%)
 x: array([  3.33983864e-12,   1.60559959e-06])
 y: array([0, 0])
======================================================================
FAIL: statsmodels.tsa.statespace.tests.test_sarimax.test_simple_time_varying
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/venv/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/venv/local/lib/python2.7/site-packages/statsmodels/tsa/statespace/tests/test_sarimax.py", line 1834, in test_simple_time_varying
    assert_almost_equal(res.params, [0,0], 6)
  File "/venv/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 452, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/venv/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 812, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/venv/local/lib/python2.7/site-packages/numpy/testing/utils.py", line 645, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 6 decimals
(mismatch 50.0%)
 x: array([  3.33983864e-12,   1.60559959e-06])
 y: array([0, 0])
```

See: https://travis-ci.org/MacPython/statsmodels-wheels/jobs/139376879

Is it reasonable to relax the precision a little?

**Comment by josef-pkt** [comment]
Yes, I think we can lower precision a bit, 

both tests have a catch_warning for convergence failure, so those cases are difficult or fragile edge cases. The second test has a followup assert that might also need adjustments.

We go intermittent failures or convergence errors on Windows 32-bit but stopped testing for it, see #3026

**Comment by matthew-brett** [comment]
Suggested relaxation at https://github.com/statsmodels/statsmodels/pull/3069

**Comment by bashtage** [comment]
Closeable?

**Comment by matthew-brett** [comment]
Still waiting for 0.8 backport?

**Comment by matthew-brett** [comment]
Backport of test precision fix : #3093

**Comment by josef-pkt** [comment]
backported
